File: scripts/windowcascade.lua
-- Command: WindowCascade

function window_cascade ()
  if context == CONTEXT_MAIN then
    init_command();
    clear_selection();
  end
  windowCascade();
  end_command();
end

[Tips]
ToolTip=&Cascade
StatusTip=Cascade the windows:  CASCADE

[Prompt]
Alias=CASCADE, WINDOWCASCADE

File: src/commands/windowcascade.c
/* Command: windowcascade */

#include "commands.h"

int window_cascade_cmd(State *state)
{
    /* NOTE: main() is run every time the command is started.
     *       Use it to reset variables so they are ready to go.
     */
    if (state->context == CONTEXT_MAIN) {
        init_command();
        clear_selection();
    }
    window_cascade();
    end_command();
    return 0;
}

